contract-examples/contracts/IRewardManager.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+import "./IAllowList.sol";
+
+interface IRewardManager is IAllowList {
+  function setRewardAddress(address addr) external;
+
+  function allowFeeRecipients() external;
+
+  function disableRewards() external;
+
+  function currentRewardAddress() external view returns (address rewardAddress);
+
+  function areFeeRecipientsAllowed() external view returns (bool isAllowed);
```

Comment:
I was thinking it would be a little bit complicated with that case:

```
setRewards(address addr, bool allowFeeRecipieints)
```

```
getRewards() (address rewardAddress, bool isAllowed)
```

also disabling it would be harder since it should be called specifically like that:
```
 setRewards(<blackhole addr>, false)
```

Could you provide what's in your mind?

Reply:
I think you're right here actually that it will be better and simpler to implement it this way as discussed offline, marking this as resolved.